File: sample/venomtyme.py
# Once upon a time...

class Cat:
  def __init__(self, props):
    self.location = props['location']
    self.birthDate = props['birthDate']
    self.currentDate = props['currentDate']
    self.catlikes = props['catlikes']

  def get_age(self):
    return self.calc_age()

  def calc_age(self):
    return self.currentDate - self.birthDate

# ...there was a cat named Katlyn

Catppuccin = Cat({
  'location': 'Cataliputra',
  'birthDate': 2016,
  'currentDate': 2022,
  'catlikes': ['Tasty Food', 'Long Naps']
})

File: sample/venomtyme.java
class Cat {
  private String location;
  private int birthDate;
  private int currentDate;
  private String[] catlikes;

  public Cat(String location, int birthDate, int currentDate, String[] catlikes) {
    this.location = location;
    this.birthDate = birthDate;
    this.currentDate = currentDate;
    this.catlikes = catlikes;
  }

  public int getAge() {
    return this.calcAge();
  }

  public int calcAge() {
    return this.currentDate - this.birthDate;
  }
}

// ...there was a cat named Katlyn
public class catppuccin {
  Cat cat = new Cat(
      "Cataliputra",
      2016,
      2022,
      new String[] { "Tasty Food", "Long Naps" });
}